samples/Sandbox/QueryStringTenantIdentificationStrategy.cs

```diff
@@ -1,27 +1,24 @@
-﻿using System;
-using System.Linq;
-using Autofac.Multitenant;
+﻿using Autofac.Multitenant;
```

Comment:
Good example of churn and larger reviews - this file didn't actually need to be touched at all to get things done.

Can you revert changes on files that didn't need to change so there's less to review here? Thanks.

Reply:
I cleanup unused imports :)